docs/site/Copyright-generator.md

```diff
@@ -0,0 +1,76 @@
+---
+lang: en
+title: 'Generate copyright/license header for JavaScript/TypeScript files'
+keywords: LoopBack 4.0, LoopBack 4
+sidebar: lb4_sidebar
+permalink: /doc/en/lb4/Copyright-generator.html
+---
+
+### Synopsis
+
+The `lb4 copyright` command runs inside a Node.js project with `package.json` to
+add or update copyright/license header for JavaScript and TypeScript files based
+on `package.json` and git history.
+
+The command also supports [lerna](https://github.com/lerna/lerna) monorepos. It
+traverses all packages within the monorepo and apply copyright/license headers.
+
+```sh
+lb4 copyright [options]
+```
+
+The following is an example of such headers.
+
+```js
+// Copyright IBM Corp. 2020. All Rights Reserved.
+// Node module: @loopback/cli
+// This file is licensed under the MIT License.
+// License text available at https://opensource.org/licenses/MIT
+```
+
+The year(s) is built from the git history of the file and `Node module` is read
+from the `name` property in `package.json`.
+
+Please note the command expects `git` is installed.
+
+### Options
+
+`--owner` : _(Optional)_ The owner of the copyright, such as `IBM Corp.`.
+
+`--license` : _(Optional)_ The name of the license, such as `MIT`.
+
+### Interactive Prompts
+
+The command prompts you for:
+
+1. The copyright owner. The default value is read from `copyright.owner` or
+   `author` in the `package.json`.
+
+2. The license name. The default value is read from `license` in the
+   `package.json`.
+
+The default owner is `IBM Corp.` and license is `MIT` with the following
```

Comment:
IIRC if it behaves the same way as strong-tools, if there's no value set for `copyright.owner` and `author`, the default owner is `Owner`.

Reply:
It says `with the following package.json` as we read values from package.json as defaults for the prompt.